src/lib/components/Logo/Logo.svelte

```diff
@@ -0,0 +1,146 @@
+  import horizLogoAvif2x from "$lib/assets/ldaf-horiz-white-rgb-1200px.png?format=avif&quality=85&imagetools";
+
+  import Image from "$lib/components/Image";
+  import type { Placement } from "./logoOptions";
+
+  const logoMap: Record<
+    Placement,
+    {
+      logo: string;
+      logo2x: string;
+      width: number;
+      height: number;
+      logoWebp: string;
+      logoWebp2x: string;
+      logoAvif: string;
+      logoAvif2x: string;
+    }
+  > = {
+    "mobile-header-main": {
+      logo: textLogo,
+      logo2x: textLogo2x,
+      width: textLogoWidth,
+      height: textLogoHeight,
+      logoWebp: textLogoWebp,
+      logoWebp2x: textLogoWebp2x,
+      logoAvif: textLogoAvif,
+      logoAvif2x: textLogoAvif2x,
+    },
+    "mobile-header-menu": {
+      logo: stackedCommissionerLogo,
+      logo2x: stackedCommissionerLogo2x,
+      width: stackedCommissionerLogoWidth,
+      height: stackedCommissionerLogoHeight,
+      logoWebp: stackedCommissionerLogoWebp,
+      logoWebp2x: stackedCommissionerLogoWebp2x,
+      logoAvif: stackedCommissionerLogoAvif,
+      logoAvif2x: stackedCommissionerLogoAvif2x,
+    },
+    "desktop-header": {
+      logo: horizWideCommissionerLogo,
+      logo2x: horizWideCommissionerLogo2x,
+      width: horizWideCommissionerLogoWidth,
+      height: horizWideCommissionerLogoHeight,
+      logoWebp: horizWideCommissionerLogoWebp,
+      logoWebp2x: horizWideCommissionerLogoWebp2x,
+      logoAvif: horizWideCommissionerLogoAvif,
+      logoAvif2x: horizWideCommissionerLogoAvif2x,
+    },
+    "mobile-footer": {
+      logo: stackedLogo,
+      logo2x: stackedLogo2x,
+      width: stackedLogoWidth,
+      height: stackedLogoHeight,
+      logoWebp: stackedLogoWebp,
+      logoWebp2x: stackedLogoWebp2x,
+      logoAvif: stackedLogoAvif,
+      logoAvif2x: stackedLogoAvif2x,
+    },
+    "desktop-footer": {
+      logo: horizLogo,
+      logo2x: horizLogo2x,
+      width: horizLogoWidth,
+      height: horizLogoHeight,
+      logoWebp: horizLogoWebp,
+      logoWebp2x: horizLogoWebp2x,
+      logoAvif: horizLogoAvif,
+      logoAvif2x: horizLogoAvif2x,
+    },
+  };
+
+  export let placement: Placement = "mobile-header-main";
+
+  const { logo, logo2x, width, height, logoWebp, logoWebp2x, logoAvif, logoAvif2x } =
+    logoMap[placement];
+</script>
+
+<!-- TODO: Replace alt value with content from CMS. -->
+<!-- The alt text for all logos should probably be the same; there probably
+     isn't much of a need to differentiate the variations of the logo to a
+     non-sighted user. -->
+<Image
+  src={logo}
+  alt="Louisiana Department of Agriculture and Forestry logo"
+  {width}
+  {height}
+  loading="eager"
+  sources={[
+    { type: "image/avif", srcset: [logoAvif, [logoAvif2x, width * 2]] },
+    { type: "image/webp", srcset: [logoWebp, [logoWebp2x, width * 2]] },
+    { type: "image/png", srcset: [logo, [logo2x, width * 2]] },
+  ]}
+/>
```

Comment:
Besides the `alt` text, this section is pretty much unchanged from the original component.